store/tikv/snapshot.go

```diff
@@ -788,5 +829,17 @@ func (rs *SnapshotRuntimeStats) String() string {
 		d := time.Duration(ms) * time.Millisecond
 		buf.WriteString(fmt.Sprintf("%s_backoff:{num:%d, total_time:%s}", k.String(), v, execdetails.FormatDuration(d)))
 	}
+	if rs.scanDetail != nil {
+		buf.WriteString(fmt.Sprintf(", rocksdb{%s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d}",
+			execdetails.ProcessKeysStr, rs.scanDetail.ProcessedKeys,
+			execdetails.TotalKeysStr, rs.scanDetail.TotalKeys,
```

Comment:
{ Process key, Total key, Process time, Total time } should be moved out of `rocksdb{}`